checkov/common/images/commons.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from collections.abc import Iterable
+from typing import TYPE_CHECKING
+
+from checkov.common.bridgecrew.check_type import CheckType
+
+if TYPE_CHECKING:
+    from checkov.common.bridgecrew.platform_integration import BcPlatformIntegration
+
+
+def enable_image_referencer(
```

Comment:
@gruebel Can this be in the existing ImageReferncer file?

Reply:
I was worried it will be a problem, because of the existing imports, but looks like it works 🚀